README.ops.md

```diff
@@ -0,0 +1,69 @@
+### Deployment on AWS
+
+To create a new environment, you'll need an RDS database first. There may be one already available or you can use the scripts in `/terraform` to help with that.
+
+First copy `terraform.tfvars.example` to `terraform.tfvars` and update passwords as needed.
+
+Then make any modifications necesary to the `.tf` files and run `terraform apply`. So long as you have `safecast` set up as an AWS profile or appropriate AWS environment variables, it should just work.
```

Comment:
We should mention about getting AWS secret key pair. Also mentioning about `AWS CLI` is nice. I think that it would be easier to create AWS profile with awscli